docker/compose/dev/install-configs.sh

```diff
@@ -15,18 +14,28 @@ if [[ ! $demo_host_address =~ ^http ]]; then
     demo_host_address="http://${demo_host_address}"
 fi
 
+# copy .env files from templates
+mkdir -p ./agent
+cp ../../.env.ssi-agent ./agent/.env.local
```

Comment:
Is there a reason why we need to have the env files in the docker dir at this point.
Can't we simply use the example env files from the packages dir for it at this point. We now have a script and the less locations we have for the same concept the better.

Reply:
fixed